src/main/java/roomescape/auth/LoginMemberArgumentResolver.java

```diff
@@ -0,0 +1,50 @@
+package roomescape.auth;
+
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletRequest;
+import java.util.Arrays;
+import org.springframework.core.MethodParameter;
+import org.springframework.web.bind.MissingRequestCookieException;
+import org.springframework.web.bind.support.WebDataBinderFactory;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.ModelAndViewContainer;
+import roomescape.auth.domain.LoginMember;
+import roomescape.auth.dto.MemberDetailResponse;
+
+public class LoginMemberArgumentResolver implements HandlerMethodArgumentResolver {
+    public static final String TOKEN_NAME = "token";
+    private final AuthService authService;
```

Comment:
```suggestion
    public static final String TOKEN_NAME = "token";
    
    private final AuthService authService;
```

여기도 개행하면 더 보기 편할 것 같네요!

Reply:
아 넵! 반영해보겠습니다~!